src/WebJobs.Script/Config/ActiveHostConfigurationSource.cs

```diff
@@ -30,17 +31,41 @@ public ActiveHostConfigurationProvider(ActiveHostConfigurationSource configurati
             }
 
             public override bool TryGet(string key, out string value)
+            {
+                if (_configurationSource?.ServiceProvider?.GetService(typeof(IConfiguration)) is IConfigurationRoot activeHostConfiguration)
+                {
+                    return
+                        (value = activeHostConfiguration.GetValue<string>(key)) != null ||
+                        (value = activeHostConfiguration.GetValue<string>(NormalizeKey(key))) != null;
+                }
+
+                value = default;
+                return false;
+            }
+
+            public override IEnumerable<string> GetChildKeys(IEnumerable<string> earlierKeys, string parentPath)
             {
                 var activeHostConfiguration = _configurationSource?.ServiceProvider?.GetService(typeof(IConfiguration)) as IConfigurationRoot;
+                var keys = new HashSet<string>();
 
                 if (activeHostConfiguration != null)
                 {
-                    value = activeHostConfiguration.GetValue<string>(key);
-                    return value != null;
+                    foreach (var config in activeHostConfiguration.GetSection(parentPath).GetChildren())
+                    {
+                        keys.Add(config.Key);
+                    }
                 }
 
-                value = default;
-                return false;
+                keys.UnionWith(earlierKeys);
+                return keys;
+            }
+
+            private static string NormalizeKey(string key)
```

Comment:
Can you describe the scenario you're trying to address by normalizing this? The limitation is already a behavior that is in place with the current configuration functionality, that behavior shouldn't change (i.e. a setting with a `:` in a different platform shouldn't suddenly start working).

Reply:
Removing NormalizeKey() here - as Brett mentioned this is only needed when reading from EnvironmentVariables directly and the EnvironmentVariableConfigurationProvider already handles both formats.